### ISSUE TITLE
Stop assigning new volunteers to districts that don't have any voters left to adopt!

### ISSUE THREAD

**Issue by sjforman** [sjforman/votefwd#184]
Story: as a new volunteer, I am automatically assigned to my geographically closest district *that still has voters available*, rather than just my geographically closest district.

**Comment by carlmanaster** [comment]
```
select d.district_id from districts d where not exists (select * from voters v where v.district_id  = d.district_id);
```
Yields TX32, KS03, CA48, TX07 in the seed data.  So any zipcode near one of them should demonstrate the problem.

CA48 zipcode is... 92660

**Comment by sjforman** [comment]
Closed by #197
